installing-the-jsx-babel-plugin.md

```diff
@@ -31,7 +31,33 @@ SystemJS.config({
 })
 ```
 
-> Alternatively we could also update our package to the `jsx` file extension here if we wish.
+> Alternatively we could also update our package to the `jsx` file extension here if we wish. (Note we updated `"main": "component.jsx",` and also renamed the actual component file.
+
+```javascript
+SystemJS.config({
+  ...
+
+  packages: {
+    "jspm-react-component": {
+      "main": "component.jsx",
+      "format": "esm",
+      "meta": {
+        "*.js": {
+          "babelOptions": {
+            "plugins": ["babel-plugin-transform-react-jsx"]
+          }
+        },
```

Comment:
Perhaps we just need to include the `"*.jsx"` part for this example. If using both, a single `*` meta may make more sense.